### ISSUE TITLE
NuGet Release

### ISSUE THREAD

**Issue by kfrancis** [lukeautry/DotNetWebSDK#1]
Are you planning on releasing on NuGet? I would love to contribute to this by either using it and providing feedback, or contributing directly here.

This project fills a need of mine to organize our POCO DTOs and API calls and it's much simpler than the alternatives.

**Comment by lukeautry** [comment]
Yeah, I'm actually packaging it as NuGet locally and consuming it that way from the sample project, but I haven't yet released it as a public NuGet package. I need to do a little more testing before I do that, but it's coming very soon.
